holoviews/plotting/bokeh/renderer.py

```diff
@@ -42,7 +51,7 @@ def __call__(self, obj, fmt=None):
                 json = plotobj.vm_serialize(changed_only=True)
                 data[plotobj.ref['id']] = {'type': plotobj.ref['type'],
                                            'data': json}
-            return serialize_json(data), {'file-ext':json, 'mime_type':MIME_TYPES[fmt]}
+            return serialize_json(data), {'file-ext':' json', 'mime_type':MIME_TYPES[fmt]}
```

Comment:
Why is there a space in the ``file-ext``? Shouldn't it be ``'json'`` not `` json``?